include/networkit/centrality/PageRank.hpp

```diff
@@ -34,17 +46,21 @@ class PageRank final : public Centrality {
      * @param[in] G Graph to be processed.
      * @param[in] damp Damping factor of the PageRank algorithm.
      * @param[in] tol Error tolerance for PageRank iteration.
+     * @param[in] distributeSinks Set to distribute PageRank values for sink nodes. (default =
+     * false)
+     * @param[in] normalized Set to normalize Page-Rank values in order to decouple it from the
+     * network size. (default = false)
      */
-    PageRank(const Graph &G, double damp = 0.85, double tol = 1e-8, bool normalized = false);
+    PageRank(const Graph &G, double damp = 0.85, double tol = 1e-8, bool distributeSinks = false,
+             bool normalized = false);
```

Comment:
This changes the meaning of a call such as `PageRank(G, 0.5, 1e-3, true)` (which now enables distributeSinks instead of normalization).

Please make `distributeSinks` the last parameter. Please also make it use an `enum class SinkHandling { NO_SINK_HANDLING, DISTRIBUTE_SINKS }` or something similar.

Reply:
Added an enum. Currently it seems, that `enum class` is not directly supported in Cython (will be supported in 0.3.0). I was not able to successfully apply any of the known workarounds. Also tried some type-casting. In the end everytime is boils down to some of those:
```
...
/Users/fbt/Documents/maks/networkit/main/fork2/networkit/centrality.cpp:39886:132: error: invalid operands to binary expression ('enum NetworKit::PageRank::SinkHandling' and 'int')
                            return (enum NetworKit::PageRank::SinkHandling) (((((enum NetworKit::PageRank::SinkHandling)digits[1]) << PyLong_SHIFT) | (enum NetworKit::PageRank::SinkHandling)digits[0]));
...
```

This also happens if other enums are changed to `enum class`.